### PR TITLE
 Improvements: Change RR Sink validator from JsonSchemaSubclassDeterminer to JsonSchemaOutputValidator

### PR DESCRIPTION
**Pull request by lciolecki**
* Removed `JsonSchemaSubclassDeterminer` instead of please use `JsonSchemaOutputValidator`
  * Added param `Value validation mode` at RR response component
  * We no longer support `nullable` param from Everit schema. Nullable schema are supported by union with null e.g. `["null", "string"]

### CODE REVIEW

File: docs/Changelog.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 * [#3709](https://github.com/TouK/nussknacker/pull/3709) Fix BestEffortJsonSchemaEncoder - fix encoding only strings when `"type": String`
 * [#3707](https://github.com/TouK/nussknacker/pull/3707) Handling apikey in OpenAPI
 * [#3716](https://github.com/TouK/nussknacker/pull/3716) Fix: JsonSchema sinks allow to add additional fields also in `strict validation mode`, if schema permits them.
+* [#3727](https://github.com/TouK/nussknacker/pull/3727) Improvements: Change RR Sink validator from JsonSchemaSubclassDeterminer to JsonSchemaOutputValidator
```
Comment (arkadius) [inline docs/Changelog.md:31]: From the user's perspective, it is a technical detail, maybe we can write some short human-friendly explanation?

**Review by arkadius — APPROVED** [review]
LGTM, please add migration for validation mode parameter

File: docs/Changelog.md [hunk 2]
```diff
@@ -28,6 +28,9 @@
 * [#3709](https://github.com/TouK/nussknacker/pull/3709) Fix BestEffortJsonSchemaEncoder - fix encoding only strings when `"type": String`
 * [#3707](https://github.com/TouK/nussknacker/pull/3707) Handling apikey in OpenAPI
 * [#3716](https://github.com/TouK/nussknacker/pull/3716) Fix: JsonSchema sinks allow to add additional fields also in `strict validation mode`, if schema permits them.
+* [#3727](https://github.com/TouK/nussknacker/pull/3727) Improvements: Change RR Sink validation way:
+  * Added param `Value validation mode` at RR response component
+  * We no longer support `nullable` param from Everit schema. Nullable schema are supported by union with null e.g. `["null", "string"]
```
Comment (gadomsky) [inline docs/Changelog.md:33]: please add this point to MigrationGuide as well

Comment (lciolecki) [reply docs/Changelog.md:33]: Is there any sens to duplicate these information?

Comment (gadomsky) [reply docs/Changelog.md:33]: I think so. I would even say - MigrationGuide is more imporant one.

MigrationGuide should contain **all of the infromation** to upgrade NU installation consciously and safely.

**Review by Filemon279 — APPROVED** [review]
LGTM :+1:

**Pull request merged**